servers/src/main/java/tachyon/worker/block/meta/BlockMeta.java

```diff
@@ -22,15 +22,18 @@
  */
 public class BlockMeta extends BlockMetaBase {
   private final long mBlockSize;
+  private long mLastAccessTimeMs;
```

Comment:
If the only purpose of this value is to show it in the UI, I would prefer to remove it for now (and not show it in the UI). The block meta class should be kept as light as possible. We can implement a separate listener that keeps track of block access times which could be reused between evictors and the UI later if this feature is very useful.